ml-agents-envs/pydoc-config.yaml

```diff
@@ -0,0 +1,25 @@
+# config to specify which modules will be used to render api docs
+folder: docs
+modules:
+    - name: mlagents_envs
+      file_name: Python-API-Documentation.md
+      submodules:
+        - env_utils
```

Comment:
I do not think env_utils, communicator, registry.binary_utils, registry.remote_registry_entry, registry.base_registry_entry and maybe more should not be a part of this.

Reply:
Ok. I'll remove.